db/migrate/20200103010349_change_notification_id_to_be_string_in_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeNotificationIdToBeStringInNotifications < ActiveRecord::Migration[5.2]
```

Comment:
Oh sick will keep that in mind for the future. I'll ask you about if there's a way to clean up the current ones too since it would make things much more organized.